Gemfile.lock

```diff
@@ -1,309 +0,0 @@
-GEM
```

Comment:
I don't think we're meant to delete this file?

Reply:
Looks like `Gemfile.lock` supposed be deleted according to the [instruction](https://docs.google.com/document/d/1eiHpZlhEDMDG8gi4LHyv2aZs1TJmeFlF8sqCE57_j4A/edit#heading=h.rz8l5qn0ygc0)